install/share/root-autobind.ldif

```diff
@@ -1,19 +0,0 @@
-# root-autobind, config
-dn: cn=root-autobind,cn=config
-changetype: add
-objectClass: extensibleObject
-objectClass: top
-cn: root-autobind
-uidNumber: 0
-gidNumber: 0
-
-dn: cn=config
-changetype: modify
-replace: nsslapd-ldapiautobind
-nsslapd-ldapiautobind: on
-
-dn: cn=config
-changetype: modify
-replace: nsslapd-ldapimaptoentries
-nsslapd-ldapimaptoentries: on
```

Comment:
Not sure if this is important but you remove this from new installs but leave it on existing ones.

Reply:
I removed the lines because lib389 configures ``nsslapd-ldapimaptoentries: on`` for us. The feature is ``on`` on existing and new systems.

Reply:
I'll have to double-check. I started with an existing install and the value remained on. I then did a fresh install and it was off. I tested by doing an ldapsearch of cn=config for each. The names work out to current version, upgraded version, fresh install with upgraded version:

```
/tmp/config:nsslapd-ldapimaptoentries: on
/tmp/config2:nsslapd-ldapimaptoentries: on
/tmp/config3:nsslapd-ldapimaptoentries: off
```

Reply:
Nice catch, Rob!